tests/utils.py

```diff
@@ -218,7 +218,7 @@ def setUp(self):
 
         # Create the datacenter
         self.database = ConsDbSchema(schema=schema, engine=engine, join_templates=joins)
-        self.data_center = DataCenter(schemas={"testdb": self.database})
+        self.data_center = DataCenter(schemas={"testdb": self.database}, user_path="")
 
     def tearDown(self) -> None:
         self.db_file.close()
```

Comment:
I'm not quite sure what to make of the GHA sadness below, nor how to comment on it on the PR.